### PR TITLE
Prevent picked up order from being updated to verified order in frontend and backend

### PR DESCRIPTION
**Pull request by jtquach1**
## Describe your changes

Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.
* Frontend: Removed View ETASU button from Picked Up Orders tab in Pharmacy view. Fixed if condition where `No orders yet.` text doesn't show up when there are no orders. Applied Prettier formatting through VS Code.
* Backend: Prevent order from being updated if its dispense status is 'Picked Up'

## Issue ticket number and Jira link
Please include the Jira Ticket Number and Link for this issue/task.
https://jira.mitre.org/browse/REMS-449

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

**Review by KeeyanGhoreshi — COMMENTED** [review]
for future reference, when making linting/prettier changes to code that is not new, make it a separate commit.  It makes it easier to tell what is actually new in the PR.

File: backend/src/routes/doctorOrders.js [hunk 1]
```diff
@@ -60,6 +60,12 @@ router.patch('/api/updateRx/:id', async (req, res) => {
     const order = await doctorOrder.findById(req.params.id).exec();
     console.log('found by id!');
 
+    console.log('order', order);
+    if (order.dispenseStatus === 'Picked Up') {
```
[the remark this answers is not in the document]
Comment (jtquach1) [reply backend/src/routes/doctorOrders.js:64]: I agree. I've got WIP changes stashed away on a local branch off of this branch that moves the dispenseStatus to an enum (it touches a lot of stuff also worth refactoring that I'd consider appropriate to be part of a bigger refactoring PR/ticket)

**Comment by jtquach1** [comment]
> for future reference, when making linting/prettier changes to code that is not new, make it a separate commit. It makes it easier to tell what is actually new in the PR.

Thanks! I'll keep that in mind. I wonder if we should have a GitHub Action that auto-applies Prettier to the files touched by the PR? Or is that handled by https://github.com/mcode/pims/actions/runs/5685884735/job/15411690590?pr=39 and https://github.com/mcode/pims/actions/runs/5685884735/job/15411691021?pr=39?

**Review by zacharyrobin — APPROVED** [review]
lgtm

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**